pkg/proxy/nftables/proxier.go

```diff
@@ -952,34 +955,34 @@ func (proxier *Proxier) syncProxyRules() {
 			loadBalancerTrafficChain = fwChain
 		}
 
-		var internalTrafficFilterTarget, internalTrafficFilterComment string
-		var externalTrafficFilterTarget, externalTrafficFilterComment string
+		var internalTrafficFilterVerdict, internalTrafficFilterComment string
+		var externalTrafficFilterVerdict, externalTrafficFilterComment string
 		if !hasEndpoints {
 			// The service has no endpoints at all; hasInternalEndpoints and
 			// hasExternalEndpoints will also be false, and we will not
 			// generate any chains in the "nat" table for the service; only
 			// rules in the "filter" table rejecting incoming packets for
 			// the service's IPs.
-			internalTrafficFilterTarget = "REJECT"
-			internalTrafficFilterComment = fmt.Sprintf(`"%s has no endpoints"`, svcPortNameString)
-			externalTrafficFilterTarget = "REJECT"
+			internalTrafficFilterVerdict = "reject"
+			internalTrafficFilterComment = fmt.Sprintf("%s has no endpoints", svcPortNameString)
+			externalTrafficFilterVerdict = "reject"
 			externalTrafficFilterComment = internalTrafficFilterComment
 		} else {
 			if !hasInternalEndpoints {
 				// The internalTrafficPolicy is "Local" but there are no local
 				// endpoints. Traffic to the clusterIP will be dropped, but
 				// external traffic may still be accepted.
-				internalTrafficFilterTarget = "DROP"
-				internalTrafficFilterComment = fmt.Sprintf(`"%s has no local endpoints"`, svcPortNameString)
+				internalTrafficFilterVerdict = "drop"
```

Comment:
given the discussion on this topic, should NFTables mode start with "reject" here?

Reply:
I can change it if you want; other than dropping support for localhost nodeports, I mostly didn't change any of the semantics relative to iptables yet.

Reply:
I'm fine with "yet"